UPGRADES.md

```diff
@@ -0,0 +1,94 @@
+2. Fill out the template fields according to the GitHub documentation for this FAQ, then select **Create Stack**. You can monitor the resource creation from the **Stacks** section of the CloudFormation service in the AWS Portal. 
+3. Wait for the resource creation process to complete for your deployed stack. 
+
+### Step 2. Upload License File
+1. Log in to the Network License Manager Dashboard.
+	- *If using a new network license manager*:<ol style="list-style-type:lower-roman"><li>In the **Outputs** tab for your new stack, locate the ```MatlabProductionServerLicenseServer``` key and click the corresponding URL. This takes you to the Network License Manager for MATLAB Dashboard login page.</li><li>Log in with the username manager and the password you specified while creating the stack. </li></ol>
+	- *If using an existing network license manager*:<ol style="list-style-type:lower-roman"><li>From the EC2 instance for your network license manager, locate the public IPv4 DNS and click the corresponding URL. This takes you to the Network License Manager for MATLAB Dashboard login page.</li><li>Log in with the previously configured username and password.</li><li>Navigate to the [reference architecture documentation](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws#deploy-reference-architecture-for-your-release) for the MATLAB release of your new deployment. Follow the additional configuration steps in the “How to use an existing license server in an existing VPC?” subsection, under “How do I use an existing VPC to deploy MATLAB Production Server?” in the FAQ section. (For R2023a and later, follow the steps under "Use Existing License Server in Existing VPC," under "Additional Information.")</li></ol>
+2. Follow the instructions in the dashboard to retrieve and upload the license file. Make sure to choose the release and operating system that match your new deployment. 
+3. (Optional) Deactivate the previous license in the License Center. 
+
+### Step 3. Migrate Applications to New Deployment
+**Note**: These instructions are for deployed applications compiled using one of the six most recent versions of MATLAB Runtime. For all other applications, refer to [Deployed Applications](UPGRADES.md#deployed-applications).
+
+1. Navigate to the S3 bucket containing the applications for your existing deployment. This can be found in the **Outputs** tab for your existing deployment's stack, with the key ```MATLABProductionServerApplicationsBucket```. 
+2. Select all code archives (``.ctf`` files) in the **auto_deploy** folder, then select **Actions > Move**. 
```

Comment:
Global comment: What convention are we using when choosing whether to use monospace font vs bold font?